python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -1012,7 +1012,13 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 
 #### Router BGP VRFs
 
-{%         if router_bgp.vrfs | selectattr('evpn_multicast', 'arista.avd.defined', true) | list %}
+{%         if router_bgp.vrfs | selectattr('evpn_multicast', 'arista.avd.defined', true) | list and router_bgp.vrfs | selectattr('graceful_restart.enabled', 'arista.avd.defined', true) | list %}
+| VRF | Route-Distinguisher | Redistribute | EVPN Multicast | Graceful Restart |
+| --- | ------------------- | ------------ | -------------- | ---------------- |
+{%         elif router_bgp.vrfs | selectattr('graceful_restart.enabled', 'arista.avd.defined', true) | list %}
+| VRF | Route-Distinguisher | Redistribute | Graceful Restart |
+| --- | ------------------- | ------------ | ---------------- |
+{%         elif router_bgp.vrfs | selectattr('evpn_multicast', 'arista.avd.defined', true) | list %}
 | VRF | Route-Distinguisher | Redistribute | EVPN Multicast |
 | --- | ------------------- | ------------ | -------------- |
 {%         else %}
```

Comment:
I would prefer if we just add the graceful restart column here unconditionally. It means you need to rerun all molecule scenarios to regenerate the docs. (`cd arista_collections/arista/avd/molecule; make refresh-facts`).

Reply:
Hi Claus,
done. 
"Graceful Restart "Column will be rendered unconditionally and I've swapped the "EVPN Multicast" Column with the "Graceful Restart" column to achieve homogenous rendering in case "EVPN Multicast" is present.

Hope this is Ok